.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- As the[ last code reviwer suggested](https://github.com/shobeee/Hello-Microverse-/pull/1#discussion_r979122138) the .gitignore file should have two files ignored. Kindly add the line of code to ignore the test.md file. :eyes: